ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/sno/test/cucushift-installer-rehearse-aws-ipi-sno-test-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+echo "************ single-node test command ************"
+
+# Tests execution
+set +e
+
+NODE_NAME=`oc get node  |grep master | awk '{print $1}'`
+Instance_Type=`aws ec2 describe-instances --filters "Name=private-dns-name,Values=${NODE_NAME}" --query "Reservations[*].Instances[*].[InstanceType]"`
+
+if [[ "${Instance_Type}" == "m6i.2xlarge" ]]; then
```

Comment:
have you checked the original install-config in your job? was the instance type configured in it?

Reply:
In fact, I have only checked the results of manual ipi installation. The instance type configured is m6i.2xlarge, but I don't know how to check the original install-config in ci job.